app/src/org/commcare/activities/PromptApkUpdateActivity.java

```diff
@@ -26,6 +45,93 @@ protected void onCreate(Bundle savedInstanceState) {
         } catch (SessionUnavailableException e) {
             // we are showing the prompt before user login, so nothing to mark
         }
+        if (toPrompt.isForced()) {
```

Comment:
If the update is not available from play core APIs (For sideloaded apps), does the old behaviour of prompts remains the same. Asking because ICDS uses this prompt functionality with CommCare mostly sideloaded on their phones and these changes should not affect those users.

Reply:
Yeah, there is no change in old behaviour, The new UI will come on top of older UI so if it fails the older functionality can take over.